### ISSUE TITLE
goog.require emitted for every complete request

### ISSUE THREAD

**Issue by thheller** [rksm/clj-suitable#6]
Sorry for posting an image, didn't know how to show it otherwise. Note that I pressed auto-complete a couple of times and each time a goog.require is done. Using the rebel-readline setup.

Seems like something that should only be done once?

![clj-suitable](https://user-images.githubusercontent.com/144930/66355909-40d43e00-e969-11e9-92a5-333461356f42.jpg)

**Comment by rksm** [comment]
Yes, this should be done only once. For repls other than shadow-cljs this is the case, I'll add a load check for shadow in the next version.

**Comment by thheller** [comment]
That example was from rebel-readline + figwheel. Didn't involve shadow-cljs at all.

**Comment by rksm** [comment]
OK, thanks, I will look into it.
